script/src/syscalls/spawn.rs

```diff
@@ -0,0 +1,266 @@
+use crate::cost_model::transferred_byte_cycles;
+use crate::syscalls::utils::load_c_string;
+use crate::syscalls::{
+    Source, SourceEntry, INDEX_OUT_OF_BOUND, SLICE_OUT_OF_BOUND, SPAWN,
+    SPAWN_EXCEEDED_MAX_CONTENT_LENGTH, SPAWN_EXCEEDED_MAX_PEAK_MEMORY, SPAWN_MAX_CONTENT_LENGTH,
+    SPAWN_MAX_MEMORY, SPAWN_MAX_PEAK_MEMORY, SPAWN_MEMORY_PAGE_SIZE, SPAWN_WRONG_MEMORY_LIMIT,
+    WRONG_FORMAT,
+};
+use crate::types::Indices;
+use crate::types::{set_vm_max_cycles, CoreMachineType, Machine};
+use crate::TransactionScriptsSyscallsGenerator;
+use crate::{ScriptGroup, ScriptVersion};
+use ckb_traits::{CellDataProvider, ExtensionProvider, HeaderProvider};
+use ckb_types::core::cell::{CellMeta, ResolvedTransaction};
+use ckb_vm::{
+    cost_model::estimate_cycles,
+    registers::{A0, A1, A2, A3, A4, A5, A7},
+    DefaultMachineBuilder, Error as VMError, Memory, Register, SupportMachine, Syscalls,
+};
+use std::sync::{Arc, Mutex};
+
+pub struct Spawn<DL> {
+    data_loader: DL,
+    group_inputs: Indices,
+    group_outputs: Indices,
```

Comment:
could you explain that why we add `group_inputs` and `group_outputs` fields since there is a `script_group` field already, aren't same with `script_group.group_inputs` and `script_group.group_outputs`?

Reply:
I think it is safe to remove these two parameters.

Reply:
> I think it is safe to remove these two parameters.

resolved by https://github.com/nervosnetwork/ckb/pull/3989 please help to review, thanks.